README.md

```diff
@@ -0,0 +1,495 @@
+# sanctuary-def
+
+sanctuary-def facilitates the definition of curried JavaScript functions which
+are explicit about the number of arguments to which they may be applied and the
+types of those arguments.
+
+```javascript
+const $ = require('sanctuary-def');
+
+const def = $.create($.env);
```

Comment:
I agree that this is unfortunate. I like to begin a README with a straightforward example, and I don't see a straightforward way to convey why it's necessary to create an "environment". Perhaps we should write this:

```javascript
const def = $.create($.env);  // Explained in "Environments" section
```

I'm open to suggestions. :)